include/envoy/upstream/cluster_manager.h

```diff
@@ -75,6 +75,24 @@ class ClusterManager {
 public:
   virtual ~ClusterManager() {}
 
+  /**
+   * Warming state a cluster is currently in.
+   */
+  enum class ClusterWarmingState : int {
+    INVALID = 0,
```

Comment:
Can you add comments to these? Also, when would you be called with INVALID?

Reply:
This was used in the unit tests for variable initialization, but it's not really needed, so I removed it.